_pkgdown.yml

```diff
@@ -18,25 +18,27 @@ reference:
       - mmrm-package
   - title: Helper Functions
     contents:
-      - h_free_cores
-      - h_mmrm_tmb_control
       - h_mmrm_tmb
   - title: Functions
     contents:
-      - component
-      - df_1d
-      - df_md
-      - fit_single_optimizer
       - mmrm
+      - mmrm_control
+      - fit_single_optimizer
       - refit_multiple_optimizers
+      - df_1d
+      - df_md
+      - component
+      - free_cores
   - title: Methods
     contents:
-      - emmeans_support
       - mmrm_methods
       - mmrm_tmb_methods
   - title: Covariance Structures
     contents:
       - covariance_types
+  - title: Estimate Marginal Means
```

Comment:
Seems a bit off-topic in this PR and without proper motivation

Reply:
Thanks @BrianLang for having a look!
In this case, since it is a small clean-up change, it is ok to include here.